src/main/java/frc/robot/subsystems/trapElvSubsystem/TrapElvSubsystem.java

```diff
@@ -309,6 +309,12 @@ private double getWristEncoderPos() {
     return Units.radiansToRotations(m_wristMotorSim.getAngleRads());
   }
 
+  public BooleanSupplier isAMPReady() {
+    double offset = wristStateGoal - getWristEncoderPos();
+    return () ->
+        (-TrapElvConstants.ROLLER_DEADZONE < offset && offset < TrapElvConstants.ROLLER_DEADZONE);
```

Comment:
I don't think this means what you think it means.  This is for the roller?  I don't see anything that was defined for wrist position

Reply:
Edited to be wrist deadzone